client/src/api/ConnectionApi.ts

```diff
@@ -3,13 +3,17 @@ import type { AxiosResponse } from 'axios'
 import { apiCall } from './BaseUrl'
 
 export const createInvitation = (agentName?: string, agentImageUrl?: string): Promise<AxiosResponse> => {
-  return apiCall.post('/connections/create-invitation', {
+  return apiCall.post('/oob/create-legacy-invitation', {
```

Comment:
Why does the `ConnectionApi` contain `oob` code? Should it not be in a `oobApi`?

Reply:
I added it to ConnectionApi for now as we only support legacy invitations. If we decide to switch to oob (or support both, see #94) we can refactor the redux store and with an `OutOfBandSlice` and all but i thought that would be overkill for now.